sasql/sasql.php

```diff
@@ -0,0 +1,685 @@
+<?php
+/**
+ * Stubs for sasql extension
+ * Taken from official SAP documenation
+ * @link     http://dcx.sap.com/index.html#sa160/en/dbprogramming/php-api.html
+ * @link     http://dcx.sap.com/sa160/en/pdf/dbprogramming16.pdf
+ * @author   Michael Moravec
+ */
+
+const SASQL_ASSOC = 0;
+const SASQL_BOTH = 0;
+const SASQL_D_INPUT = 0;
+const SASQL_D_OUTPUT = 0;
+const SASQL_INPUT_OUTPUT = 0;
+const SASQL_NUM = 0;
+const SASQL_STORE_RESULT = 0;
+const SASQL_USE_RESULT = 0;
```

Comment:
I believe we need to extract the actual values from the extension distribution binaries. Otherwise, it may lead to errors like "duplicate array keys" when constants are used as array keys.

Reply:
Yes, I don't have access to any SA installation/extension though. :/ So at least better than udefined constants ATM.

Reply:
Let me see if I can extract them during the day. This is where I currently am:
```
↪ ldd php-5.6.0_sqlanywhere.so
        statically linked

↪ php -dextension=$PWD/php-5.6.0_sqlanywhere.so
Warning: PHP Startup: The SQLAnywhere client libraries could not be loaded. Please ensure that libdbcapi.so can be found in your LD_LIBRARY_PATH environment variable. in Unknown on line 0
```
Going to try the Windows DLL later.

Reply:
From the extension [source](https://archive.sap.com/documents/docs/DOC-40537):
```c
REGISTER_LONG_CONSTANT("SASQL_D_INPUT", 1, CONST_CS | CONST_PERSISTENT);
REGISTER_LONG_CONSTANT("SASQL_D_OUTPUT", 2, CONST_CS | CONST_PERSISTENT);
REGISTER_LONG_CONSTANT("SASQL_D_INPUT_OUTPUT", 3, CONST_CS | CONST_PERSISTENT);

REGISTER_LONG_CONSTANT("SASQL_USE_RESULT", 0, CONST_CS | CONST_PERSISTENT);
REGISTER_LONG_CONSTANT("SASQL_STORE_RESULT", 1, CONST_CS | CONST_PERSISTENT);

REGISTER_LONG_CONSTANT("SASQL_NUM", 1, CONST_CS | CONST_PERSISTENT);
REGISTER_LONG_CONSTANT("SASQL_ASSOC", 2, CONST_CS | CONST_PERSISTENT);
REGISTER_LONG_CONSTANT("SASQL_BOTH", 3, CONST_CS | CONST_PERSISTENT);
```

Reply:
Cool, updated!